src/contrib/cluster/Akka.DistributedData/WriteAggregator.cs

```diff
@@ -87,50 +89,58 @@ protected override void PreStart()
             if (IsDone) Reply(isTimeout: false);
         }
 
-        protected override bool Receive(object message) => message.Match()
-            .With<WriteAck>(x =>
-            {
-                Remaining = Remaining.Remove(SenderAddress);
-                if (IsDone) Reply(isTimeout: false);
-            })
-            .With<WriteNack>(x =>
-            {
-                _gotNackFrom = _gotNackFrom.Add(SenderAddress);
-                if (IsDone) Reply(isTimeout: false);
-            })
-            .With<DeltaNack>(_ =>
-            {
-                // ok, will be retried with full state
-            })
-            .With<UpdateSuccess>(x =>
-            {
-                _gotLocalStoreReply = true;
-                if (IsDone) Reply(isTimeout: false);
-            })
-            .With<StoreFailure>(x =>
-            {
-                _gotLocalStoreReply = true;
-                _gotNackFrom = _gotNackFrom.Remove(_selfUniqueAddress.Address);
-                if (IsDone) Reply(isTimeout: false);
-            })
-            .With<SendToSecondary>(x =>
+        protected override bool Receive(object message)
```

Comment:
Practically the same, modernizing Match()...With() with a switch instead.

Reply:
LGTM